java/src/test/java/org/rocksdb/util/BytewiseComparatorTest.java

```diff
@@ -242,14 +242,22 @@ private void doRandomIterationTest(
             result_iter.seekToLast();
             break;
           case 2: {
-            // Seek to random key
+            // Seek to random (existing or non-existing) key
             final int key_idx = rnd.nextInt(source_strings.size());
-            final String key = source_strings.get(key_idx);
+            final String key = source_strings.get(key_idx) + (rnd.nextBoolean() ? "" : "non-existing");
```

Comment:
I am not sure I understand the purpose of:
`final String key = source_strings.get(key_idx) + (rnd.nextBoolean() ? "" : "non-existing");`

Also I don't think that `rnd.nextBoolean()` can ever return an empty string, so I am not sure this does what you intended.